docs/dictionary-and-concepts/dictionary-metering.md

```diff
@@ -12,43 +12,61 @@ The terms listed here are all terms relating to the topics of defining metering.
 | Accounting Point |  A domain under balance responsibility where Energy Supplier change can take place and for which commercial business processes are defined. **Additional information:** This is a type of Metering Point. |
 | Asset Type | The technology used on a specific metering point  |
 | **B** |   |
+| Business Reason Code | Indicates the business context, e.g. E23 = 'Periodic metering' used with time series |
 | **C** |   |
+| Created Date Time | A date and time property used in all messages exchanged with Green Energy Hub. It indicates when a particular message was created, and it is provided by the sender of the message |
```

Comment:
I think we are missing some context information. I think it is important that this prop is used in the context of the document of the message. The same term will most likely be used for other purposes in other contexts.

This probably applies to a number of other terms too.

Reply:
Created Date Time is generic property, and I believe the use of it will be quiet similar, if not the same, in different contexts. 
I did consider not adding this when I made the PR, and given your comment I will remove it and add it to the out-of-scope-generic-terms, mentioned in the PR description. Same goes for Industry Classification.